kinto_client/batch.py

```diff
@@ -15,7 +16,7 @@ def request(self, method, endpoint, data=None, permissions=None,
         # is called.
         self.requests.append((method, endpoint, data, permissions, headers))
         # This is the signature of the session request.
-        return None, None
+        return defaultdict(dict), defaultdict(dict)
```

Comment:
this conflicts with #34. I prefer the other approach, returning constants for batched responses here.